### ISSUE TITLE
Are there any plan to support the --cclk and --mclk for AMD GPU

### ISSUE THREAD

**Issue by otcagon** [Lolliedieb/lolMiner-releases#1461]
I am not sure current update version are no issue of crash. 
The previos version 1.46a sometimes shows crash and even the reset key is not working.
So, the power should be off and restart the windows10.
The radeon software should be running and set the defined core and memory clock.
Otherwise the maximum clock applied.
After then lolminer batch shuld be executed.

If is support the clock option for AMD (I am using RX6600). it make me more comfortable.

Thank you

**Comment by jgonzis** [comment]
> I am not sure current update version are no issue of crash. The previos version 1.46a sometimes shows crash and even the reset key is not working. So, the power should be off and restart the windows10. The radeon software should be running and set the defined core and memory clock. Otherwise the maximum clock applied. After then lolminer batch shuld be executed.
> 
> If is support the clock option for AMD (I am using RX6600). it make me more comfortable.
> 
> Thank you

You are completely right. that is an interesting feature. Problem is that normally Drivers update of AMD changes also that, that is constantly update. We know that normally users use the Adrenaline Driver or the Linux Mining OS.

**Comment by kadiev75** [comment]
Please make these features work, now I can't manage all my AMD cards !!!
